README.md

```diff
@@ -1,2 +1,170 @@
 # orchestrator-helm-chart
-Helm chart to deploy the Orchestrator solution suite, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
+Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator, Knative Eventing, Knative Serving, Data Index and Job Service.
+
+This chart will deploy the following on the target OpenShift cluster:
+  - Janus IDP backstage
+  - SonataFlow Operator
+  - OpenShift Serverless Operator
+  - Knative Eventing
+  - Knative Serving
+  - Data Index Service
+  - Job Service
+  - PostgreSQL Databases 
+  - Sample workflow (event-timeout)
+
+## Installation
+Build helm dependency:
+```console
+$ git clone git@github.com:parodos-dev/orchestrator-helm-chart.git
+$ cd orchestrator-helm-chart/charts
+$ helm dep update orchestrator
+```
+Perform a first pass install:
+```console
+$ helm install orchestrator orchestrator
+NAME: orchestrator
+LAST DEPLOYED: Thu Nov  2 09:02:29 2023
+NAMESPACE: jianrong
+STATUS: deployed
+REVISION: 1
+```
+Wait for about 1 minute, and then check to make sure the SonataFlow operator and OpenShift Serverless operator have started.
+```console
+$ oc wait --for=condition=Available deploy sonataflow-operator-controller-manager -n openshift-operators
+deployment.apps/sonataflow-operator-controller-manager condition met
+$ oc wait --for=condition=Available deploy knative-openshift -n openshift-serverless
+deployment.apps/knative-openshift condition met
```

Comment:
this command fails to me:
```
→ oc wait --for=condition=Available deploy knative-openshift -n openshift-serverless
Error from server (NotFound): deployments.apps "knative-openshift" not found
```

the operator is there:
```
→ oc get operators -A
NAME                                       AGE
serverless-operator.openshift-serverless   8m57s
sonataflow-operator.openshift-operators    8m57s
```
the namespace is there as well:
```
→ oc get ns openshift-serverless
NAME                   STATUS   AGE
openshift-serverless   Active   7m44s
```
----------
Update: works, but it took several minutes to complete and for the error to be gone.
Similar to the previous command.

Reply:
OK